static/skin/viewer.js

```diff
@@ -347,6 +349,12 @@ function onClickEvent(e) {
   if (target !== null && "href" in target) {
     const target_href = getRealHref(target);
     const target_url = new URL(target_href, iframeDocument.location);
+    if ( target_url.href.startsWith(`${FULL_ROOT_URL}/viewer#`) &&
+         !linkShouldBeOpenedInANewWindow(target, e) ) {
+      contentIframe.contentWindow.parent.location = target_url;
+      e.preventDefault();
+      return;
```

Comment:
Just to be sure: JS can stack a list of event handlings to the same event. It is really important that the rest of the stack of events already associated to the mouse click continues to be handled like before. Can you please confirm this?